### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by ldlshizhu** [swz30/MPRNet#26]
Hello, thank you for your contribution. 
I changed batchsize to 1 (in training.yml), and used two GPUs to run, but the loss was about 1200 in the first two epochs, and turned into INF in the third epoch. Could  you please tell me why? (I am running debluring)

**Comment by ldlshizhu** [comment]
Hello, thank you for your patience!
Your suggestion is very effective for me. After changing the learning rate to 1.5e-4, the loss was about 100 in the first epoch and dropped to 70 in the 69th epoch, but it suddenly became 800,000 in the 70th epoch! I don't know what happened…

**Comment by swz30** [comment]
Hi @ldlshizhu 

If you run codes with the default settings, there should not be an issue. 

Try even lower learning rate: 1e-4.

**Comment by ldlshizhu** [comment]
> Hi @ldlshizhu
> 
> If you run codes with the default settings, there should not be an issue.
> 
> Try even lower learning rate: 1e-4.

Thank you very much! I'll try again!
